src/main/java/com/adobe/target/sample/service/TargetClientService.java

```diff
@@ -43,16 +43,18 @@ public TargetDeliveryResponse getPageLoadTargetDeliveryResponse(HttpServletReque
                                                                     HttpServletResponse response) {
 
         Context context = getContext(request);
-
+        PrefetchRequest prefetchRequest = getPrefetchRequest();
         ExecuteRequest executeRequest = new ExecuteRequest();
         RequestDetails pageLoad = new RequestDetails();
         executeRequest.pageLoad(pageLoad);
-
+        executeRequest.addMboxesItem(new MboxRequest().index(0).name("Happy-clientHints"));
```

Comment:
Let's name both these mboxes something more generic since customers will use this as a reference